database/README.md

```diff
@@ -0,0 +1,168 @@
+Apply the Service definition to the cluster:
+```bash
+kubectl apply -f ps-service.yaml -n [namespace]
+service/postgres created
+```
+* Check the database infrastructure.
+```bash
+kubectl get all -n [namespace]
+NAME                            READY   STATUS    RESTARTS   AGE
+pod/postgres-65db968757-ncnvk   1/1     Running   0          54s
+
+NAME                 TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)          AGE
+service/kubernetes   ClusterIP   10.96.0.1       <none>        443/TCP          8m53s
+service/postgres     NodePort    10.96.248.178   <none>        5432:31047/TCP   19s
+
+NAME                       READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/postgres   1/1     1            1           54s
+
+NAME                                  DESIRED   CURRENT   READY   AGE
+replicaset.apps/postgres-65db968757   1         1         1       54s
+```
+
+* Test the connection to the Database.
+  
+Identify the PostgreSQL pod.
+```bash
+kubectl get pods -n [namespace]
+NAME                        READY   STATUS    RESTARTS   AGE
+postgres-65db968757-ncnvk   1/1     Running   0          2m38s
+```
+
+```bash
+kubectl exec -it [postgres_pod] -- psql -h localhost -U [db_user] --password -p 5432 [db_name]
+```
+Replace the `postgres_pod`, `db_user` and `db_name` with the values from `postgres-cm.yaml`. The `--password` prompts for the password interactively.
+```bash
+kubectl exec -it postgres-65db968757-ncnvk -- psql -h localhost -U ps_user --password -p 5432 postgresdb
+Password: 
+psql (16.3 (Debian 16.3-1.pgdg120+1))
+Type "help" for help.
+
+postgresdb=# 
+```
+### Create a table and populate the Database
+
+The script `init_db.go` will create a table in the Database and will insert the test objects from `test_objects.sql`
+* Add the Database access configuration. The values are the same from the `postgres-cm.yaml`.
+```go
+const (
+	host     = "localhost"
+	port     = 5432
+	user     = // the db_user from postgres-cm.yaml, ie: "admin", "ps_user".
+	password = // the db_password from postgres-cm.yaml
+	dbname   = // the db_name from postgres-cm.yaml, ie: "postgres", "test_db".
+)
+```
+* Run the script
+```bash
+go run init_db.go
+table test_objects created in the DB.
+testdata from test_objects.sql inserted in the DB.
+```
+## SQL queries (examples)
+These queries can be executed from the Database.
+Connect to the Database.
+```bash
+kubectl exec -it [postgres_pod] -- psql -h localhost -U [db_user] --password -p 5432 [db_name]
+```
+```sql
+SELECT * FROM test_objects;
+```
+```sql
+SELECT * FROM test_objects WHERE kind='Job';
+```
+```sql
+SELECT * FROM test_objects WHERE data::jsonb->>'kind'='Job';
+```
+## Configure DBeaver
+* Create a port-forward.
+```bash
+kubectl port-forward -n [namespace] svc/postgres 5432:5432
+```
+* DBeaver settings
+  Go to `Database` > `New Database Connection`.
+  Select `PostgreSQL`.
+  Fill the connection parameters with the values from `postgres-cm.yaml`. And click on the `Finish` button.
```

Comment:
Since it is a separate sentence, you don't need "And" here.

```suggestion
  Fill the connection parameters with the values from `postgres-cm.yaml`. Click on the `Finish` button.
```